src/backend/base/langflow/components/jigsawstack/text_to_sql.py

```
@@ -0,0 +1,88 @@
+                params["database"] = self.database
+            if self.file_store_key:
+                params["file_store_key"] = self.file_store_key
+    
+            response = client.text_to_sql(params)
+            
+            if not response.get("success", False):
+                raise ValueError("JigsawStack API returned unsuccessful response")
+            
+            return Data(data=response)
+            
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)